module1_solution/app.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+(function() {
+
+	angular.module('LunchCheck', [])
+		.controller('LunchCheckController', LunchCheckController);
+
+	LunchCheckController.$inject = ['$scope'];
+
+	function LunchCheckController($scope) {
+
+		$scope.remarks = '';
```

Comment:
use controllerAs pattern https://github.com/johnpapa/angular-styleguide/tree/master/a1#controllers

Read these two articles, 
https://johnpapa.net/do-you-like-your-angular-controllers-with-or-without-sugar/

https://daveceddia.com/convert-scope-to-controlleras/
In particular, make sure you understand the "Behind the Scene" Section that explains name spacing